### ISSUE TITLE
[Item] Symbol of Life

### ISSUE THREAD

**Issue by RustZenith** [dalaranwow/dalaran-wow#5384]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If deleted, Symbol of life [item=6866] impossible to retrieve. Making Redemption an unlearnable skill to Paladins.
**Expected behaviour**: Tell us what should happen instead
Race specific NPC's to give if missing. Duthorian Rall [npc=6171] for Humans, Tiza Battleforge [npc=6179] for Dwarfs.
**Steps to reproduce the problem**:

1. Destroy Symbol of Life [item=6866], With The Tome of Divinity [quest=1786] in questlog.

**Include proofs for this behaviour**
https://wowgaming.altervista.org/aowow/?quest=1790
![wowscrnshot_040418_210224](https://user-images.githubusercontent.com/37970684/38346708-cc75af78-384b-11e8-8a66-cf99b2532fdc.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
This is not exactly an issue for the issue tracker unless the NPC in question is supposed to give you a new Symbol of Life and you didn't provide proof of this being the case. First drop the quest and pick it up again. If this doesn't help, then make an in-game ticket. Recommend closing this issue as invalid.

**Comment by RustZenith** [comment]
I literally linked the quest in which he gives the replacement https://wowgaming.altervista.org/aowow/?quest=1790, and it wont work

**Comment by Chenocide** [comment]
Try abandoning the quest an retake it. You will most likely be waiting a long time for this to be fixed. 

#1565 already mentioned that a new Symbol of Life is not given out, not sure how related it is.
